tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1548,6 +1548,10 @@ in the loop variable.
     for x in [5, 7, 11]:
         statement # Loop iterates 3 times with 'x' as 5, then 7 and finally 11.
 
+    var arr: Array[String] = ["a", "b", "c"]
+    for s: String in arr: # Typed loop variable.
```

Comment:
This is correct, but the type is redundant here: it is inferred since the array is typed. The main benefit of this feature is that it allows you to downcast the element type, like `for control: Control in get_children()`. Also, this is useful for dictionaries that do not currently support typing.

```suggestion
    var arr: Array = ["a", "b", "c"]
    for s: String in arr: # Typed loop variable.
```

It's already documented [Static typing in GDScript](https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/static_typing.html#specify-the-element-type-of-an-array):

https://github.com/godotengine/godot-docs/blob/881b736a6f846d56aa4993d25994c075004c5603/tutorials/scripting/gdscript/static_typing.rst?plain=1#L218-L226

Could we just duplicate this?